--- a/video/decode/vd_lavc.c
+++ b/video/decode/vd_lavc.c
@@ -138,7 +138,7 @@
         .skip_frame = AVDISCARD_DEFAULT,
         .framedrop = AVDISCARD_NONREF,
         .dr = -1,
-        .hwdec_api = (char *[]){"no", NULL,},
+        .hwdec_api = (char *[]){"yes", NULL,},
         .hwdec_codecs = "h264,vc1,hevc,vp8,vp9,av1,prores",
         // Maximum number of surfaces the player wants to buffer. This number
         // might require adjustment depending on whatever the player does;
